target/linux/generic/backport-5.15/050-v5.16-1-mips-uasm-Enable-muhu-opcode-for-MIPS-R6.patch

```diff
@@ -24,9 +31,11 @@ Signed-off-by: Johan Almbladh <johan.almbladh@anyfinetworks.com>
  Ip_u3u1u2(_nor);
  Ip_u3u1u2(_or);
  Ip_u2u1u3(_ori);
+diff --git a/arch/mips/mm/uasm-mips.c b/arch/mips/mm/uasm-mips.c
+index 7154a1d99aad..e15c6700cd08 100644
```

Comment:
Please refresh the patches like this: `make target/linux/{clean,refresh} V=99` 
quilt will remove these two lines.